lib/OpenQA/Task/Asset/Download.pm

```diff
@@ -25,18 +25,36 @@ sub register {
     $app->minion->add_task(download_asset => \&_download);
 }
 
+sub _create_symlinks {
+    my ($job, $ctx, $assetpath, $other_destinations) = @_;
+
+    my @error_message;
+    for my $link_path (@$other_destinations) {
+        $ctx->debug(qq{Creating symlink "$link_path" to "$assetpath"});
+        symlink($assetpath, $link_path) || push @error_message,
+          "Cannot create symlink from $assetpath to $link_path: $!";
+    }
+    if (scalar(@error_message) > 0) {
+        $ctx->error(my $msg = join('\n', @error_message));
```

Comment:
`join('\n', …)` needs to use `"\n"` instead to properly evaluate the control character

Reply:
Good catch. Fixed as separate commit because I've just copied that code.